AppServices/Validators/CreateCustomerRequestValidator.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Application.DTOs;
+using FluentValidation;
+using System;
+
+namespace Alan_WarrenDesafio1.Validators
+{
+    public class CreateCustomerRequestValidator : AbstractValidator<CreateCustomerRequest>
+    {
+        public CreateCustomerRequestValidator()
+        {
+            RuleFor(x => x.FullName)
+                .NotEmpty()
+                .WithMessage("Full name must not be null or empty")
+                .MinimumLength(2)
+                .WithMessage("Full name length must be more than 2 digits")
+                .MaximumLength(300)
+                .WithMessage("Full name length must cannot have more than 50 digits");
+
+            RuleFor(x => x.Email)
+                .NotEmpty()
+                .WithMessage("Email must not be null or empty")
+                .MinimumLength(9)
+                .WithMessage("Email length must be more than 9 digits")
+                .MaximumLength(256)
+                .WithMessage("Email length must cannot have more than 256 digits")
+                .Must(y => y.IsValidEmail())
+                .WithMessage("Email is invalid, please try again");
```

Comment:
FluentValidation escreve excelentes mensagens de erros, validações simples não precisamos sobrescrever mensagens de erros
```suggestion
            RuleFor(x => x.Email)
                .NotEmpty()
                .MinimumLength(9)
                .MaximumLength(256)
                .Must(y => y.IsValidEmail());
```